chapter5.Rmd

```diff
@@ -10,14 +10,14 @@ You may remember from the chapter about matrices that all the elements that you
 
 When doing a market research survey, however, you often have questions such as:
 
-- 'Are your married?' or 'yes/no' questions (`logical`)
+- 'Are you married?' or 'yes/no' questions (`logical`)
 - 'How old are you?' (`numeric`)
 - 'What is your opinion on this product?' or other 'open-ended' questions (`character`)
 - ...
 
 The output, namely the respondents' answers to the questions formulated above, is a data set of different data types. You will often find yourself working with data sets that contain different data types instead of only one. 
 
-A data frame has the variables of a data set as columns and the observations as rows. This will be a familiar concept for those coming from different statistical software packages such as SAS or SPSS.
+A data frame has the variables of a data set as columns and the observations as rows. An R data frame is the same concept as an SQL database table, or a SAS dataset, or a MATLAB table, or an SPSS table.
```

Comment:
Seems like the most obvious analogy would be an Excel spreadsheet. Many people won't be familiar with these other languages/environments